addons/hr_fleet/models/employee.py

```diff
@@ -14,13 +14,12 @@ def action_open_employee_cars(self):
         self.ensure_one()
         cars = self.env['fleet.vehicle.assignation.log'].search([
             ('driver_id', 'in', (self.user_id.partner_id | self.sudo().address_home_id).ids)]).mapped('vehicle_id')
-
         return {
             "type": "ir.actions.act_window",
             "res_model": "fleet.vehicle",
             "views": [[False, "kanban"], [False, "form"], [False, "tree"]],
             "domain": [("id", "in", cars.ids)],
-            "context": dict(self._context, create=False),
+            "context": dict(self.env.context, create=False, group_by=[]),
```

Comment:
How do i retrieve the action in this method ?

Reply:
If I am not mistaken, the web client will not interpret this context as its new context, but instead as the action-specific part of the context.  It should combine the general context (language, timezone, company) with the new action-specific context.  So this one could be
```suggestion
            "context": dict(create=False),
```
One should check that nothing critical is lost with this.

Reply:
@rco-odoo Yes, general context will get completed with action-specific one.
But indeed you are losing the context defined in the action in xml.